apps/vendor-portal/src/modules/orders/pages/orders-edit.vue

```diff
@@ -315,6 +315,49 @@ export default defineComponent({
       order,
       items: computed(() => order.value?.items),
       loading,
+      shippingInfo: computed(
+        (): {
```

Comment:
Do we need to use inline type/interface here? Maybe it is better to move interface declaration in root ./types.